FE/angular/src/app/components/movie-card/movie-card.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="image col-3"><img [src]="movie.Poster" [alt]="movie.Title" /></div>
```

Comment:
Is there a way to better handle the situation when a `movie.Poster` is not found? Currently, a broken image is displayed.

Reply:
Yes, I could have add a default image in case of error on image load.